cmake/bob_robotics.cmake

```diff
@@ -144,7 +146,7 @@ macro(BoB_project)
     # When using the ARSDK (Parrot Bebop SDK) the rpath is not correctly set on
     # Ubuntu (and presumably when linking against other libs in non-standard
     # locations too). This linker flag fixes the problem.
-    if(UNIX)
+    if(UNIX AND NOT APPLE)
```

Comment:
Thanks, this looks like a handy reference. It doesn't matter here for now because nobody's flying drones from a mac but there's a bunch of ``rpath`` hackery lying around so it'll def bite us at some point.